docs/source/filter.mdx

```diff
@@ -0,0 +1,12 @@
+# Filter rows in a dataset
+
+Datasets Server provides a `/filter` endpoint for filtering rows in a dataset.
+
+<Tip warning={true}>
+  Currently, only <a href="./parquet">datasets with Parquet exports</a>
+  are supported so Datasets Server can index the contents and run the search without
+  downloading the whole dataset.
+</Tip>
+
+This guide shows you how to use Datasets Server's `/filter` endpoint to filter rows based on a query string.
```

Comment:
Nice to have prepared the docs. It would be nice to explain more (here or in openapi.json) the values / syntax accepted for `query`

Reply:
Done.